src/main/java/org/jboss/xavier/integrations/route/filter/HeaderFilter.java

```diff
@@ -9,7 +9,7 @@
 
     @Override
     public boolean applyFilterToCamelHeaders(String arg0, Object arg1, Exchange arg2) {
-        return false;
+        return true;
```

Comment:
This method seems to be saying we want to apply filters to Headers but in 
src/main/java/org/jboss/xavier/integrations/route/MainRouteBuilder.java configure() we've made a change to say 

> .to("http4://oldhost?headerFilterStrategy=#noFilter")

which doesn't seem to tie up

> .to("http4://oldhost?headerFilterStrategy=#noFilter")

Reply:
Good catch. In fact the #noFilter is the name of the bean. So to be consistent with the name we should not do any filtering.
Let me check :+1:

Reply:
Done